src/dataframe/dataframe.jl

```diff
@@ -51,7 +51,9 @@ All columns in `columns` must be `AbstractVector`s and have the same length.
 An exception are `DataFrame(kwargs...)` and `DataFrame(pairs::Pair{Symbol,<:Any}...)`
 form constructors which additionally allow a column to be of any other type that is not
 an `AbstractArray`, in which case the passed value is automatically repeated to fill
-a new vector of the appropriate length.
+a new vector of the appropriate length. As a particular rule if a column is
+`Ref` or `0`-dimentional `AbstractArray` then it is allowed and in both cases
```

Comment:
I'd just say "`Ref` or `0`-dimensional `AbstractArray` values are treated in the same way."

Reply:
fixed in both places